client/controllers/electionCtrl.js

```diff
@@ -148,5 +161,9 @@ module.exports = [
     $scope.isChosen = function (alternative) {
       return $scope.priorities.includes(alternative);
     };
+
+    $scope.reset = function () {
+      location.reload();
```

Comment:
Why work hard when you can `location.reload()`? :brain: :shipit:

Reply:
🙉